### PR TITLE
REP-I4: update Motoman-specific messages

### PR DESCRIPTION
**Pull request by ted-miller**

### CODE REVIEW

File: rep-I0004.rst [hunk 1]
```diff
@@ -284,23 +284,29 @@ Motoman
...
+      2011  ROS_MSG_MOTO_IOCTRL_REPLY             -
 
- 2011-2015  -                                     Reserved for future use
+ 2012-2015  -                                     Reserved for future use
 
-      2016  ROS_MSG_MOTO_JOINT_TRAJ_PT_FULL_EX    -
-      2017  ROS_MSG_MOTO_JOINT_FEEDBACK_EX        -
+      2016  ROS_MSG_MOTO_JOINT_TRAJ_PT_FULL_EX    Command multiple control-groups with a single message
+      2017  ROS_MSG_MOTO_JOINT_FEEDBACK_EX        Get feedback position of multiple control-groups with a single message
+      2018  ROS_MSG_MOTO_SELECT_TOOL              Set the active tool mass properties (for PFL function)
```
Comment (gavanderhoorn) [inline rep-I0004.rst]: Does this message only set tool mass? Or also other properties (tcp transform, inertia, etc)?

File: rep-I0004.rst [hunk 2]
```diff
@@ -284,23 +284,29 @@ Motoman
...
 
- 2018-2099  -                                     Reserved for future use
+ 2019       -                                     Reserved for future use
```
Comment (gavanderhoorn) [inline rep-I0004.rst]:
```suggestion
      2019  -                                     Reserved for future use
```

**Comment by gavanderhoorn** [comment]
This duplicates some parts fo #25, but that never got a review.

File: rep-I0004.rst (hunk 1 above)
Comment (ted-miller) [reply rep-I0004.rst]: All properties. I have changed the description

**Comment by gavanderhoorn** [comment]
Thanks @ted-miller :+1:

**Pull request merged**